src/spawn.cpp

```diff
@@ -90,11 +100,15 @@ bool Spawns::loadFromXml(const std::string& filename)
 					centerPos.y + pugi::cast<uint16_t>(childNode.attribute("y").value()),
 					centerPos.z
 				);
-				uint32_t interval = pugi::cast<uint32_t>(childNode.attribute("spawntime").value()) * 1000;
-				if (interval > MINSPAWN_INTERVAL) {
-					spawn.addMonster(nameAttribute.as_string(), pos, dir, interval);
+				int64_t interval = pugi::cast<int64_t>(childNode.attribute("spawntime").value()) * 1000;
```

Comment:
Inconsistent variable types. Either make everything int32_t, or int64_t (preferably the former... I don't think anyone will ever make a spawn with interval over 136 years)